src/test/java/no/unit/nva/bare/authorizer/LambdaAuthorizerTest.java

```diff
@@ -0,0 +1,145 @@
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class LambdaAuthorizerTest implements WithEnvironment {
+
+    public static final String CORRECT_SECRET_NAME = "someSecretName";
+    public static final String CORRECT_SECRET_KEY = "someSecretKey";
+    public static final String CORRECT_SECRET_VALUE = "someSecretValue";
+    public static final String AWS_SERCRETS_MANAGER_ERROR_MESSAGE = "AwsSercretsManager message";
+    public static final String DEFAULT_ENV_VALUE = "*";
+    public static final int SINGLE_STATEMENT_EXPECTED = 1;
+    public static final int UNIQUE_STATEMENT = 0;
+    public static final String ACTION_ALLOWED = "Allow";
+    public static final String ACTION_DENIED = "Deny";
+
+    public static final String DEFAULT_METHOD_ARN = "arn:aws:execute-api:eu-west-1:884807050265:2lcqynkwke/Prod/GET"
+            + "/service/users/orestis@unit.no";
+    public static final String METHOD_ARN_REQUEST_FIELD = "methodArn";
+    public static final String WRONG_SECRET_NAME = "WrongSecretName";
+    private static final String WRONG_SECRET_KEY = "WrongSecretKey";
+
+    private final Environment envWithCorrectValues;
+    private Context context;
+
+    public LambdaAuthorizerTest() {
+        secretsManager();
+        Map<String, String> correctEnvValues = Map.of(LambdaAuthorizer.AWS_SECRET_NAME_ENV_VAR, CORRECT_SECRET_NAME,
+                LambdaAuthorizer.AWS_SECRET_KEY_ENV_VAR, CORRECT_SECRET_KEY);
+        envWithCorrectValues = mockEnvironment(correctEnvValues, DEFAULT_ENV_VALUE);
+        context = mock(Context.class);
+    }
+
+    @Test
+    void shouldReturnAcceptPolicyWhenSecretIsCorrect() throws IOException {
+
+        AuthorizerResponse response = sendRequest(envWithCorrectValues);
+
+        List<StatementElement> statements = response.getPolicyDocument().getStatement();
+        assertThat(statements.size(), is(equalTo(SINGLE_STATEMENT_EXPECTED)));
+
+        StatementElement statement = statements.get(UNIQUE_STATEMENT);
+        String actualEffect = statement.getEffect();
+        assertThat(actualEffect, is(equalTo(ACTION_ALLOWED)));
+    }
+
+    @Test
+    void shouldReturnDenyPolicyWhenSecretNameIsWrong() throws IOException {
+        Map<String, String> wrongSecretName = Map.of(
+                LambdaAuthorizer.AWS_SECRET_NAME_ENV_VAR, WRONG_SECRET_NAME,
+                LambdaAuthorizer.AWS_SECRET_KEY_ENV_VAR, CORRECT_SECRET_KEY);
+
+        Environment envWithWrongSecretName = mockEnvironment(wrongSecretName, DEFAULT_ENV_VALUE);
+        AuthorizerResponse response = sendRequest(envWithWrongSecretName);
+
+        List<StatementElement> statements = response.getPolicyDocument().getStatement();
+        assertThat(statements.size(), is(equalTo(SINGLE_STATEMENT_EXPECTED)));
+
+        StatementElement statement = statements.get(UNIQUE_STATEMENT);
+        String actualEffect = statement.getEffect();
+        assertThat(actualEffect, is(equalTo(ACTION_DENIED)));
+    }
```

Comment:
I think these tests belong to NVA-commons since they do not actually test logic written in this class.